plugins/ufm_consumer_plugin/conf/ufm_consumer_plugin.conf

```
@@ -0,0 +1,4 @@
+[common]
+port_number=8997
+ssl_apache_port=7443
+apache_port=8888
```

need to add an option to set the authentication server port as well:
```suggestion
apache_port=8888
auth_service_port=8889
```